app/marathons/page.tsx

```diff
@@ -0,0 +1,140 @@
+"use client";
+
+import {
+	Container,
+	Row,
+	Col,
+	Button,
+	Card,
+	Accordion,
+	ListGroup,
+} from "react-bootstrap";
+import externalSourceData from "../../public/info/external-sources.json";
+import { EXTERNAL_SOURCE_KEYS } from "../consts/external-source-keys.const";
+
+export default function Page() {
+	const marathonExternal = externalSourceData.find(
```

Comment:
As I learned from @Kompanel 😄 - use states to reduce calling `find()` method
```suggestion
	const [marathonExternal ] = useState<ExteralSource>(externalSourceData.find((x) => x.key ==EXTERNAL_SOURCE_KEYS.BIALYSTOK_WEB)
```

**Remember to import useState and other types like in [Honorary Patronage page](https://github.com/Sztab-WOSP-Bialystok/wosp-staff/tree/main/app/home/sections/honorary-patriage)**